### ISSUE TITLE
Performance on custom medical data

### ISSUE THREAD

**Issue by MariaKokshaikina** [facebookresearch/dino#123]
Hello, thank you for this great repository. I used it to test DINO’s performance in detecting Pneumothorax in chest x-rays and segmenting tissues in histopathological images. Together with my colleague we tried different hyper parameters and configurations, however, in all our experiments activation maps showed no signs of recognising the disease.

We published the results of our experiments in this blog - https://medium.com/@mllabucu/transformer-based-self-supervised-learning-for-medical-images-41395d069829 

The tested changes include:
* Architectures of bit transformers were trained as well as regular CNN (resnext50) 
* Custom crop method to increase a chance of including region of interest in local crops
* Changed various hyper parameters (number of crops, learning rate, warmup configuration)
* Decreased dimensionality of last embedding layer since our data has less classes and is less diverse than ImageNet

We wonder if the reasons for poor performance are hidden solely in the nature of our dataset (described in the blog) or is there anything we could have missed when training DINO.

**Comment by woctezuma** [comment]
Maybe relevant:
> Cherti, Mehdi, and Jenia Jitsev. "Effect of large-scale pre-training on full and few-shot transfer learning for natural and medical images." [arXiv preprint arXiv:2106.00116](https://arxiv.org/abs/2106.00116) (2021). [(code)](https://github.com/SLAMPAI/large-scale-pretraining-transfer)

**Comment by seekingdeep** [comment]
@woctezuma so the question remains, can Dino be applied to the medical imaging field, for tasks such as nuclei image segmentation inwhich the objects of interest are densely packed together and overlapping

**Comment by seekingdeep** [comment]
@woctezuma so the question remains, can Dino be applied to the medical imaging field, for tasks such as **nuclei image segmentation** and **anomaly detection**, inwhich the object of interest is small, overlapping and dense.
![0](https://user-images.githubusercontent.com/71519088/160100779-64b475ed-0b9f-4fc5-ab59-9df6efb700f1.gif)
![1](https://user-images.githubusercontent.com/71519088/160100798-1369bb07-4492-4b9b-8a21-cd01e405db85.gif)
